paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/NewPaymentMethodVerticalLayoutUI.kt

```diff
@@ -13,11 +15,16 @@ internal fun NewPaymentMethodVerticalLayoutUI(
     paymentMethods: List<SupportedPaymentMethod>,
     selectedIndex: Int,
     isEnabled: Boolean,
+    onViewMorePaymentMethods: () -> Unit,
     onItemSelectedListener: (SupportedPaymentMethod) -> Unit,
     imageLoader: StripeImageLoader,
     modifier: Modifier = Modifier,
 ) {
     Column(modifier = modifier, verticalArrangement = Arrangement.spacedBy(12.dp)) {
+        TextButton(onClick = { onViewMorePaymentMethods() }) {
```

Comment:
once we display a saved PM on this screen, that saved PM will have a button "View more" which should have this `onClick` behavior